dist/pom.xml

```diff
@@ -247,5 +253,56 @@
     </plugins>
   </build>
 
+  <profiles>
+    <profile>
+      <id>smoke-test</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.apache.maven.plugins</groupId>
+            <artifactId>maven-failsafe-plugin</artifactId>
+            <configuration>
+              <!-- prevent forking otherwise zeebe test listner is not found in classpath -->
+              <forkMode>never</forkMode>
+            </configuration>
+          </plugin>
+          <plugin>
+            <groupId>com.bazaarvoice.maven.plugins</groupId>
+            <artifactId>process-exec-maven-plugin</artifactId>
+            <version>0.7</version>
+            <executions>
+                <!--Start process broker-->
```

Comment:
Just a very quick note: you can easily spin up containers via Testcontainers. So as long as we've built the image then there shouldn't be any issue with running a normal container based test.

Reply:
Running Containers doesn't really make sense here as we want to check that the broker runs on the different operating systems

Reply:
That's true, don't know what I was thinking :smile: